cli/kubectl-hyperkube

```diff
@@ -122,8 +123,10 @@ def remove(ctx, cluster_to_remove):
 
 @cli.command()
 @click.option('--cluster', '-g', multiple=True)
+@click.option('--k8s-config', '-k', default=f"{Path.home()}/.kube/config", required= False)
+@click.option('--show-only', '-s', is_flag=True, default= False, required= False)
```

Comment:
Why `show-only` ? It is hard to understand what that means. Maybe `dry-run`  or something else?

Reply:
ok changing it to dry-run